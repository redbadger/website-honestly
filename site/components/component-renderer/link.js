// @flow
import * as React from 'react';
import styles from './styles.css';

type LinkProps = {
  children?: React.Node,
  href: string
}

export default function Link({ children, href }: LinkProps) {
  return (
    <a className={styles.a} href={href}>
      {children}
    </a>
  );
}
