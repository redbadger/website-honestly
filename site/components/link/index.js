// @flow
import React, { PropTypes } from 'react';
import { NavigationLink } from 'navigation-react';

type LinkProps = {[id: string]: any}
export default function Link(props: LinkProps) {
  const linkProps = Object.assign({}, props, {
    stateKey: props.to,
  });

  return (
    <NavigationLink
      {...linkProps}
    >
      {props.children}
    </NavigationLink>
  );
}

Link.propTypes = {
  to: PropTypes.string.isRequired,
  children: PropTypes.node.isRequired,
};
