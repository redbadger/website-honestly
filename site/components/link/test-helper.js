// @flow

import { StateNavigator, HTML5HistoryManager } from 'navigation';
import * as React from 'react';
import { NavigationHandler } from 'navigation-react';
import makeAppNavigator from '../../routes';

export function mockNavigator(parentKey?: string = 'foo') {
  // The constructor is checked in prop-types,
  // so we use it as base and then mutate

  const navigator = new StateNavigator(
    [
      { key: 'foo', route: 'foo' },
      { key: 'bar', route: 'bar' },
      { key: 'barChild', route: 'bar/child', parentKey: 'bar' },
    ],
    new HTML5HistoryManager(),
  );

  navigator.navigate(parentKey);

  return navigator;
}

type Props = {
  stateNavigator?: StateNavigator,
  children: React.Node,
};

const initStateNavigator = () => {
  const navigator = makeAppNavigator();

  navigator.navigateLink('/');

  return navigator;
};

export const Context = (props: Props) => {
  const { children, stateNavigator = initStateNavigator() } = props;

  return <NavigationHandler stateNavigator={stateNavigator}>{children}</NavigationHandler>;
};
