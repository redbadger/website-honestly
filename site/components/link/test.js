// @flow
import React from 'react';
import { shallow } from 'enzyme';
import Link from '.';


describe('components/link', () => {
  it('renders OK children', () => {
    shallow(<Link to="foo">Hello</Link>);
  });
});
