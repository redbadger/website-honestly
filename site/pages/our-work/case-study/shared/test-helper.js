// @no-flow
import React from 'react';

import styles from './what-to-read-next/style.css';
import Link from '../../../../components/link';
import WhatToReadNextSlice from './what-to-read-next/slice';

const MockedWhatToReadNext = () => {
  const nextSlices = [
    {
      name: 'Pride in London',
      tagline: 'Delivering a best-in-class digital experience',
      image: 'pride.gif',
      linkKey: 'prideCaseStudy',
    },
    {
      name: 'Financial Times',
      tagline: 'Lasting change for a media giant',
      image: 'financialTimesReadMore.jpg',
      linkKey: 'financialTimesCaseStudy',
    },
    {
      name: 'Car Trawler',
      tagline: 'Creating complete CMS control for travel brands',
      image: 'carTrawler.jpg',
      linkKey: 'carTrawlerCaseStudy',
    },
  ];
  return (
    <div className={styles.whatNext}>
      <div className={styles.whatNext__tilesContainer}>
        {nextSlices.map((slice, index) => (
          /* disabled because list is static */
          /* eslint-disable react/no-array-index-key */
          <WhatToReadNextSlice
            key={`${slice.name}-${index}`}
            name={slice.name}
            tagline={slice.tagline}
            image={slice.image}
            linkKey={slice.linkKey}
          />
          /* eslint-enable react/no-array-index-key */
        ))}
      </div>
      <Link to="ourWorkPage" className={styles.whatNext__button}>
        See more of our work
      </Link>
    </div>
  );
};

export default MockedWhatToReadNext;
