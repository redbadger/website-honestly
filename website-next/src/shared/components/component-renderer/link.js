import React, { Component } from 'react';
import styles from './styles.css';

export default class Link extends Component {
  static propTypes = {
    children: React.PropTypes.node,
    href: React.PropTypes.string.isRequired,
  };

  render() {
    return (
      <a className={styles.a} href={this.props.href}>{this.props.children}</a>
    );
  }
}
