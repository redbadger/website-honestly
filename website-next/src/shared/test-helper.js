import TestUtils from 'react-addons-test-utils';
import equals from 'lodash/isEqual';
import ShallowTestUtils from 'react-shallow-testutils';

export function render(component) {
  const renderer = TestUtils.createRenderer();
  renderer.render(component);
  return renderer.getRenderOutput();
}

export function hasProp(type, value) {
  return (component) => component.props && component.props[type] === value;
}

export function isEqual(value) {
  return (component) => equals(value, component);
}

export function findOne(predicate, component) {
  const elem = ShallowTestUtils.findAll(component, predicate);

  if (elem.length !== 1) {
    throw new Error('Did not find exactly one match');
  }

  return elem[0];
}

export function containsOne(value, component) {
  return !!findOne(isEqual(value), component);
}

export function findWithType(type, component) {
  return ShallowTestUtils.findWithType(component, type);
}

export function findAllWithType(type, component) {
  return ShallowTestUtils.findAllWithType(component, type);
}
