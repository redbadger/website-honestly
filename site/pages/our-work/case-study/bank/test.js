import React from 'react';
import { render } from 'enzyme';

import { Context } from '../../../../components/link/test-helper';
import MockedWhatToReadNext from '../shared/test-helper';
import CaseStudyBank from './index';

jest.mock('../shared/what-to-read-next', () => MockedWhatToReadNext);

describe('site/pages/our-work/case-study/bank/', () => {
  it('renders correctly', () => {
    expect(
      render(
        <Context>
          <CaseStudyBank />
        </Context>,
      ),
    ).toMatchSnapshot();
  });
});
