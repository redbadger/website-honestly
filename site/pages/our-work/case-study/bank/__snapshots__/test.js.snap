// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/our-work/case-study/bank/ renders correctly 1`] = `
Array [
  <div
    class="header"
  >
    <picture>
      <source
        class="header__image"
        srcset="header.gif"
      />
      <img
        alt="Stick figures building a large arrow from small square blocks"
        class="header__image"
        src="header.gif"
      />
    </picture>
  </div>,
  <div
    class="body"
  >
    <div
      class="heading"
    >
      <h1
        class="fontL "
      >
        Digital transformation at one of the world’s largest banks
      </h1>
    </div>
    <div
      class="section"
    >
      <h2
        class="sectionHeading"
      >
        <span
          class="redTitle"
        >
          Let’s make things better.
        </span>
        Increasing customer satisfaction & conversion rates
      </h2>
      <div
        class="sectionBody"
      >
        <p
          class="fontXS paragraph"
        >
          As part of a global digital transformation programme, the bank had a vision to design, build, support and continuously evolve the digital opportunities, products, features, platforms and capabilities from inception to customer delivery, across all markets.
        </p>
        <p
          class="fontXS paragraph"
        >
          With conversion rates at less than 5%, the Originations and Utilities programme identified problems with the customer onboarding experience of new account openings, credit card and loan applications, and approached us to help them improve it. Three of the key issues were the requirement for customers to visit bank branches, the heavy back office data entry for staff and the resulting long lead times. With dispersed teams, rigid processes and legacy technology contributing to the limited functionality of their websites, their delivery pipeline and release process needed to change.
        </p>
        <div
          class="content__float content__float--noBottomMargin"
        >
          <picture>
            <source
              class="tickImage"
              srcset="tick.png"
            />
            <img
              alt="Stick figures building a tick from small boxes"
              class="tickImage"
              src="tick.png"
            />
          </picture>
        </div>
        <p
          class="fontXS paragraph"
        >
          We identified a number of opportunities to help them become customer-centric rather than process-centric and reduce waste with Lean Agile principles, so we embarked on a large scale efficiency programme with Red Badger leading the global programme of work to deliver complex technology solutions across the whole organisation.
        </p>
      </div>
    </div>
    <div
      class="section"
    >
      <h2
        class="sectionHeading"
      >
        <span
          class="redTitle"
        >
          Do the right thing. Do the thing right.
        </span>
        Implementing customer-centricity with new ways of working
      </h2>
      <div
        class="sectionBody"
      >
        <p
          class="fontXS paragraph"
        >
          Our role was to lead the Origination and Utilities programme which had to:
        </p>
        <ul
          class="list"
        >
          <li>
            Deliver quality digital products to their customers quickly
          </li>
          <li>
            Vastly improve the customer experience of opening a new current account and applying for a loan or credit card
          </li>
          <li>
            Help them build internal capability and change the culture, leaving a legacy beyond our involvement
          </li>
          <li>
            Reduce operational costs and increase business efficiency
          </li>
        </ul>
        <p
          class="fontXS paragraph"
        >
          There was a need to change the way they worked. We implemented cross-functional teams, combined with lean processes to drive agility which allowed us to deliver value to customers quickly. This created a robust delivery model which has since been scaled to multiple global teams. At the time of writing, we have scaled from one to twelve global teams with more teams planned over the next year.
        </p>
        <h3
          class="fontS2 "
        >
          Setting up for success
        </h3>
        <p
          class="fontXS paragraph"
        >
          As part of the project validation process, we helped create the delivery plans for three product roadmaps that could be delivered by small, cross-functional teams from start to finish. Products were allocated per country starting with account opening, quickly followed by credit cards and loans. Each product team consisting of UX, Design, Engineering and an Agile Delivery Lead became accountable for the on-time success of the product into that market.
        </p>
        <h3
          class="fontS2 "
        >
          Putting the customer at the centre of the project
        </h3>
        <p
          class="fontXS paragraph"
        >
          The bank’s UXD teams, which sat separately from Originations and Utilities, had already started designing prototypes which created an opportunity to bring them into the project programme. In order to demonstrate our thinking, we ran a number of workshops to help build trust, as this was a new approach.
        </p>
        <div
          class="content__float content__float--left"
        >
          <picture>
            <source
              class="smileyImage"
              srcset="smiley.png"
            />
            <img
              alt="Stick figures building a smiley from small boxes"
              class="smileyImage"
              src="smiley.png"
            />
          </picture>
        </div>
        <p
          class="fontXS paragraph"
        >
          Rather than design prototypes, when often an idea is designed that can’t be built, we focused on designing live products to capture details of integration with external systems and requirements. This enabled live user testing and ensured complicated system requirements weren’t missed. It also allowed us to bring accessibility testing into the core delivery process, avoiding the risk of it failing at the point of release. All of these principles and practices have allowed us to deliver user value more quickly, and help build a platform where we could easily iterate on the live product.
        </p>
        <h3
          class="fontS2 "
        >
          Choosing the right tech for the job
        </h3>
        <p
          class="fontXS paragraph"
        >
          Our objective was to help the business move faster by creating more autonomy within the teams, so we set about reducing their dependencies on shared platforms and pipelines, so that we could first create, and then scale excellence.
        </p>
        <p
          class="fontXS paragraph"
        >
          The bank has a cloud-centric strategy and Red Badger are pivotal in helping them to realise their ambitions in this area. We have been working on a solution that leverages recent advances in container orchestration, combined with automated provisioning of immutable infrastructure, to enable teams to deploy their Microservice-based applications onto cloud-provider-agnostic “Microplatforms”. The teams can now create and destroy identical environments in minutes, empowering them to manage their own deployments for testing (and soon for production). This puts the teams in control so that they can deliver continuously, whilst reducing the risk of failure and the mean-time-to-recovery when failures occur.
        </p>
        <p
          class="fontXS paragraph"
        >
          By allowing the teams more freedom to align around tools and technologies, we’re already dramatically increasing velocity. Open source tools have helped speed up the process with components shared across teams for reuse. This, combined with a deep sense of ownership, means that our DevOps focused teams have full responsibility for design, build, and run (including 24x7 support).
        </p>
        <h3
          class="fontS2 "
        >
          Tech we used
        </h3>
        <p
          class="fontXS paragraph"
        >
          React, Node, Spring Boot, Go, Microservices, Jenkins, Docker in Swarm Mode, Kubernetes, AWS, GCP, Terraform
        </p>
      </div>
    </div>
    <div
      class="section"
    >
      <h2
        class="sectionHeading"
      >
        <span
          class="redTitle"
        >
          Creating lasting change.
        </span>
        Scaling excellence and increasing speed to market
      </h2>
      <div
        class="sectionBody"
      >
        <p
          class="fontXS paragraph"
        >
          So far we have launched multiple live products in both the UK and Hong Kong, with new live products launching in other global markets in 2018.
        </p>
        <p
          class="fontXS paragraph"
        >
          The results to date include the successful delivery of the new credit card application journey in Hong Kong, implementing straight-through processes, allowing customers to open an account without the need to visit a branch for the first time. This makes them the first bank in Hong Kong able to accept customers at the end of an online-only application, giving them a crucial competitive advantage.
        </p>
        <p
          class="fontXS paragraph"
        >
          We have made groundbreaking progress with the opening of current accounts with an improvement in conversion rates for Account Opening from 5% of customers completing the application to 43% since launch.
        </p>
        <div
          class="content__float"
        >
          <picture>
            <source
              class="pyramidImage"
              srcset="pyramid.png"
            />
            <img
              alt="Stick figures building a pyramid from small boxes"
              class="pyramidImage"
              src="pyramid.png"
            />
          </picture>
        </div>
        <p
          class="fontXS paragraph"
        >
          As we move forward, cross product sharing is enabling faster development times and greater consistency as components and page elements can be reused and scaled. With product teams sharing their knowledge across the whole programme we’re helping to eliminate inefficiency.
        </p>
        <p
          class="fontXS paragraph"
        >
          We’ve successfully integrated UX and Design into the fast-moving delivery environment and the projects have benefited at each stage of the process from business requirements to test and deployment, transforming the way the bank’s own UXD team work.
        </p>
        <p
          class="fontXS paragraph"
        >
          We’ve helped shift their ways of working to set them up for continuous delivery. In originations, we have thus far helped them to deploy multiple times a day to the testing environment, deploying weekly to production and releasing to customers monthly. We are moving closer and closer to true continuous deployment, providing them with the technical capability to deploy and release to customers multiple times per day. The contribution to evolving the release process with the global brand and other departments such as legal and compliance, to cater for agile product delivery is a particular highlight. This shift toward true continuous deployment has started to impact other parts of the business outside of originations, that had previously released just a few times a year.
        </p>
        <p
          class="fontXS paragraph"
        >
          Starting off with one team, we now lead more than 150 people across three countries consisting of both bank employees and third parties, including two teams in Krakow that we’ve helped onboard. By embedding our delivery and technical directors, we’ve been able to drive change and work closely with stakeholders across the globe.
        </p>
        <div
          class="quotation quote"
        >
          <div
            class="quotation__text"
          >
            <blockquote
              class="quotation__quote"
            >
              Delivering value to customers at this speed and scale requires that each cross-functional team is as self-sufficient as possible, having maximum aligned autonomy. By embracing DevOps culture and using high levels of automation, we are enabling the teams to own every aspect of their domain, from idea conception, through design and build, to runtime management and support.
            </blockquote>
          </div>
          <div
            class="quotation__author"
          >
            <div
              class="avatar__container"
              style="width:70px;height:70px;min-width:70px;min-height:70px"
            >
              <img
                alt="Stuart Harris"
                src="quote-author.jpg"
              />
            </div>
            <div
              class="quotation__author__wrapper"
            >
              <div
                class="quotation__author__name"
              >
                — Stuart Harris
              </div>
              <div
                class="quotation__author__title"
              >
                Red Badger CIO
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="section"
    >
      <div
        class="sectionBody"
      >
        <div
          class="Card listBoxContainer"
        >
          <div
            class="listBox listBox"
          >
            <h3
              class="listBox__heading"
            >
              What we did
            </h3>
            <ul>
              <li
                class="listBox__element listBox__element--results listBox__item"
              >
                <div>
                  Collaborated with the client and other consultancies
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item"
              >
                <div>
                  Embedded the bank’s resources within Red Badger cross-functional teams
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item"
              >
                <div>
                  Supported the client’s hiring process to bring best in class people into their team
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item"
              >
                <div>
                  Drove alignment and consistent ways of working across the programme
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item"
              >
                <div>
                  Led the project delivery - accountable & responsible
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item"
              >
                <div>
                  Introduced quality at scale
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item"
              >
                <div>
                  Implemented agile engineering practices
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item"
              >
                <div>
                  Introduced autonomous cross-functional teams across multiple products
                </div>
                <div />
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      class="section"
    >
      <div
        class="sectionBody"
      >
        <div
          class="Card listBoxContainer"
        >
          <div
            class="listBox listBox"
          >
            <h3
              class="listBox__heading"
            >
              Outcomes
            </h3>
            <ul>
              <li
                class="listBox__element listBox__element--results listBox__item listBox__item--outcomes"
              >
                <div>
                  Continuous deployment capability and a new release process delivering features to production
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item listBox__item--outcomes"
              >
                <div>
                  Product delivered in 12 months enabling an end to end account opening journey
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item listBox__item--outcomes"
              >
                <div>
                  Product delivered enabling straight-through processing of Hong Kong Credit Cards
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item listBox__item--outcomes"
              >
                <div>
                  Reusable components and page elements enabling cross-product sharing, increasing efficiency and velocity
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item listBox__item--outcomes"
              >
                <div>
                  UXD embedded in the end to end delivery process
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item listBox__item--outcomes"
              >
                <div>
                  Digital capability built across product, design, tech and delivery
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item listBox__item--outcomes"
              >
                <div>
                  Global transformation programme delivery - Poland and Canada (9 teams)
                </div>
                <div />
              </li>
              <li
                class="listBox__element listBox__element--results listBox__item listBox__item--outcomes"
              >
                <div>
                  Two cross-functional teams onboarded in Krakow
                </div>
                <div />
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    class="whatNext"
  >
    <div
      class="whatNext__tilesContainer"
    >
      <div
        class="whatNext__tile"
      >
        <figure>
          <a
            class=""
            href="/our-work/case-study/pride"
          >
            <img
              alt="Pride in London"
              class="whatNext__image"
              src="pride.gif"
            />
          </a>
          <figcaption
            class="whatNext__caption"
          >
            Pride in London
          </figcaption>
        </figure>
        <a
          class="whatNext__link"
          href="/our-work/case-study/pride"
        >
          <h2
            class="whatNext__title"
          >
            Delivering a best-in-class digital experience
            <span
              class="whatNext__arrow_wrapper"
            >
              <svg
                class="whatNext__arrow"
                viewBox="0 0 22 22"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11 1l10 10-10 10-1.754-1.754 6.959-7.018H1V9.772h15.205l-6.96-7.018z"
                  fill="#FC1D42"
                  fill-rule="evenodd"
                  stroke="#FC1D42"
                />
              </svg>
            </span>
          </h2>
        </a>
      </div>
      <div
        class="whatNext__tile"
      >
        <figure>
          <a
            class=""
            href="/our-work/case-study/financial-times"
          >
            <img
              alt="Financial Times"
              class="whatNext__image"
              src="financialTimesReadMore.jpg"
            />
          </a>
          <figcaption
            class="whatNext__caption"
          >
            Financial Times
          </figcaption>
        </figure>
        <a
          class="whatNext__link"
          href="/our-work/case-study/financial-times"
        >
          <h2
            class="whatNext__title"
          >
            Lasting change for a media giant
            <span
              class="whatNext__arrow_wrapper"
            >
              <svg
                class="whatNext__arrow"
                viewBox="0 0 22 22"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11 1l10 10-10 10-1.754-1.754 6.959-7.018H1V9.772h15.205l-6.96-7.018z"
                  fill="#FC1D42"
                  fill-rule="evenodd"
                  stroke="#FC1D42"
                />
              </svg>
            </span>
          </h2>
        </a>
      </div>
      <div
        class="whatNext__tile"
      >
        <figure>
          <a
            class=""
            href="/our-work/case-study/car-trawler"
          >
            <img
              alt="Car Trawler"
              class="whatNext__image"
              src="carTrawler.jpg"
            />
          </a>
          <figcaption
            class="whatNext__caption"
          >
            Car Trawler
          </figcaption>
        </figure>
        <a
          class="whatNext__link"
          href="/our-work/case-study/car-trawler"
        >
          <h2
            class="whatNext__title"
          >
            Creating complete CMS control for travel brands
            <span
              class="whatNext__arrow_wrapper"
            >
              <svg
                class="whatNext__arrow"
                viewBox="0 0 22 22"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11 1l10 10-10 10-1.754-1.754 6.959-7.018H1V9.772h15.205l-6.96-7.018z"
                  fill="#FC1D42"
                  fill-rule="evenodd"
                  stroke="#FC1D42"
                />
              </svg>
            </span>
          </h2>
        </a>
      </div>
    </div>
    <a
      class="whatNext__button"
      href="/our-work"
    >
      See more of our work
    </a>
  </div>,
  <section
    class="contactUsContainer"
    id="contactUs"
  >
    <h2
      class="header"
    >
      We can help you
    </h2>
    <div
      class="contentContainer"
    >
      <ul
        class="list"
      >
        <li
          class="item"
        >
          Create & validate new ideas
        </li>
        <li
          class="item"
        >
          Deliver great quality products & services, fast
        </li>
        <li
          class="item"
        >
          Be bold with technology
        </li>
        <li
          class="item"
        >
          Be more customer centric
        </li>
        <li
          class="item"
        >
          Improve efficiency with lean practices
        </li>
        <li
          class="item"
        >
          Build capability & confidence
        </li>
      </ul>
      <div
        class="imgContainer"
      />
    </div>
    <div>
      <a
        class="mailToLink"
        href="mailto:hello@red-badger.com?Subject=Can%20you%20help%20me%20with%20..."
      >
        Send an email
      </a>
      <span
        class="talkToUs"
      >
        hello@red-badger.com
      </span>
    </div>
  </section>,
]
`;
